### ISSUE TITLE
Redirect Request for the form 21-0958 and detail landing page

### ISSUE THREAD

**Issue by jenniferlee-dsva** [department-of-veterans-affairs/va.gov-team#8396]
### Instructions
- Requests for redirects need to be submitted **AT LEAST** 2 weeks in advance.  Some redirects take significant amount of time and coordination to implement, so start the process as soon as you know you will need one. 
- This issue will be used from initial request through implementation of the redirect to ensure all individuals working on this are notified of status updates.  Please do not create multiple issues to track differnt steps.
- It is your responsibility to notify VA stakeholders of pending redirect.  

### Description
*This is a detailed description of the request. It should include background information and reason for the request.*

- NODs are no longer accepted because the appeals process has  changed to Decision Reviews. Per form manager, Kayce White: This form has been deprecated, and the form has been removed from the form database. 
- However the legacy form detail landing page continues to live and link ppl to a 404.
- This form landing page is also one of the top 30 high organic search value form pages. (We had originally captured this as a candidate for the MVP form detail landing page product, before it got deprepcated.) Therefore, it might be appropriate to permanently redirect this detail landing page to the new Descion Reviews hub landing page (or to the Manage legacy appeals page?), where users can get contextual information about the appeals process having changed. 

- Form detail lp URL  www.va.gov/vaforms/form_detail.asp?formno=21-0958
- Form URL  http://www.vba.va.gov/pubs/forms/VBA-21-0958-ARE.pdf

### Requestor team info
*Indicate what team and who the primary contact on that team is* Jen Lee

### Implementation date
Date new URL(s) will be live: *enter date new pages/URLs will be live* Any time

Please indicate when the redirect(s) needs to be implemented"
- [ ] On the same date the new URLs launch
- [ ] Can happen within 1-5 business days after new URLs launch
- [X] Other - Please indicate timing : Any time - the legacy form is already deprecated 

**The requesting team is responsible for notifying the group working on this issue if the target date changes. They are also responsible for ensuring the destination URLs are implemented correctly and live at the time the redirects are deployed.**

### Redirects needed

Current URL  |  Redirect Destination (new URL)
---  |  ---
~http://www.vba.va.gov/pubs/forms/VBA-21-0958-ARE.pdf~ | ~https://www.va.gov/decision-reviews/legacy-appeals/~
https://www.va.gov/vaforms/form_detail.asp?formno=21-0958 | https://www.va.gov/decision-reviews/legacy-appeals/#a-legacy-appeal-follows-these-

### Definition of done
- [ ] Above information is provided and issue is tagged and assigned appropriately - *@ requesting team*
- [ ] All appropriate VA stakeholders are notified of pending redirect - *@ requesting team*
- [x] Redirect request is vetted and documented and implementation plan is clear - *@ Content & IA team*
- [x] Request is assigned to appropriate team for implementation - *@ Content & IA team*
- [ ] Implementation team completes work - *@ Implementation team*
- [ ] Implementation pushed live and redirect is validated in production - *@ all*
- [ ] Ticket is closed - *@ requesting team*

**Comment by mnorthuis** [comment]
@jenniferlee-dsva The decision review hub page could be rather confusing for someone who is looking for the NOD. The page doesn't mention that term, so there's no hints as to what a user should do.  If this particular form has that much value, I'd like to find a place to send them that helps the user understand what the new process is.   

Some ideas:
- a new content page that specifically talks about the NOD and why it is no longer available and then links them to the decision review hub page
- a question within the decision review FAQ page about the NOD

What are your thoughts?

**Comment by jenniferlee-dsva** [comment]
Good point. What if we pointed to this page https://www.va.gov/decision-reviews/legacy-appeals/ which discusses NOD no longer being used, and provides links to the new way? 

If that would be still unclear, then I think adding an NOD question in the decision review FAQs would be better than a whole page just on NOD. 

RE search value - yes, this form was on our list of forms to do 1:1 redirect to the modern form detail URL bc of the high search value. (Before it got deprecated by VBA.) 


www.va.gov/vaforms/form_detail.asp?formno=21-0958 | VA Form 21-0958 | 28,435 ysv

**Comment by mnorthuis** [comment]
I agree a whole page on NOD is likely overkill, and the legacy page is a good option, however it doesn't quite answer to the NOD very clearly for any users  being directed there.   That may be an easy add to the legacy appeal page to help clarify?

Also, I noticed there's a link in that legacy appeals page that refers to learning about the decision review process but takes them to the hub page, which is not really for learning.   Would it make sense to have a tier 2/resource page that just talks to the change of the process that we can take users to and then guide them to either manage their legacy appeals or guide them into the DR process or FAQs?   That may be overkill too...just feels like we are missing some guidance here.

**Comment by jenniferlee-dsva** [comment]
Yeah, for sure.  The tricky thing is that NODs are not accepted but ppl can use either the Supplemental Claim or other DR options to file a disagreement -- it depends on the date of the decision they're disagreeing with. And even during a legacy appeal, they can disagree with a later decision than the original decision. 

Thinking about this - it seems like... we should redirect the NOD form and form detail URLs to the Manage legacy page https://www.va.gov/decision-reviews/legacy-appeals/ AND: 

1/ the Manage legacy appeals page needs to be more explicit somewhere  about the NOD form no longer being used -- and that Suppliemental Claim and DR process are the replacements. It actually does say this on step 1, but it is so so very nuanced that it's easy to not grasp. 

and possibly also:

2/ The FAQs question https://www.va.gov/decision-reviews/faq/#how-do-i-opt-in-if-i-want-my-l 
could (should) be updated to better reflect the fact that NOD's are no longer accepted, and that users have to use Supplemental Claim or other DR options depending on the date of the decision they're disagreeing with. 

> If you disagree with a VA decision you received before February 19, 2019, you can opt in to the modernized review system after you’ve received a VA Statement of the Case (SOC) or Supplemental Statement of the Case (SSOC). To get either a SOC or an SSOC, **you must file a Notice of Disagreement (VA Form 21-0958) within 1 year of the date on your decision.**  `<--and this bold part should be updated possibly to be more explicit - NODs are no longer accepted, and that "For new disagreements with decisions dated before February 19, 2019, you'll need to file a Supplemental Claim. For disagreements with decisions dated February 19, 2019, or later, you'll need to use the new decision review process.'? `

> Find out how to file a Notice of Disagreement  `<-- this links to the Manage legacy appeals page; the text should be more around learning about the legacy appeals process, not NOD...?`

For the content/copy part, I'm going to defer to Laura and Danielle. At this point Laura is the content SME about these, more than me.

**Comment by mnorthuis** [comment]
This sounds good.  Once the content/copy decisions are made, let's come back to this to define what the final redirect looks like.

**Comment by jenniferlee-dsva** [comment]
@lwalsh617 ^ for context regarding the redirecting of 21-0958 NOD form and form detail URLs - need your content recommendations in order to finalize where/how redirect.

**Comment by lwalsh617** [comment]
@mnorthuis @jenniferlee-dsva - 
Here's my recommended edit to step 1 on the legacy page:
<img width="747" alt="Screen Shot 2020-04-30 at 9 56 25 AM" src="https://user-images.githubusercontent.com/53535663/80719110-01b63780-8ac9-11ea-8fb1-d41cd8bd65d8.png">

**Comment by lwalsh617** [comment]
@mnorthuis @jenniferlee-dsva -- Also, FWIW, I'm hugely in favor of creating a tier 2 decision reviews learning page per Mikki's comments above. We really need a place to spell out old versus new process.

**Comment by jenniferlee-dsva** [comment]
@lwalsh617 - Interesting. The concept of a learning page distinguishing the new vs. old -- it seems like that is what the "Manage legacy appeals" page is trying to do with the steps and the forking ppl to the new process. 

Also - I don't know this for sure, but it seems like for ppl who want to file a **brand new appeal**, we would only confuse them if we told them what the 'old' way was. For these ppl, they don't need to know anything about the 'old' way -- it is closed to them and basically doesn't exist for them. 

So in that sense, it's hard to see what the value of learning about the old process is. 

We only need to guide ppl through what the old process is if they have an existing older claim that's in flight via the legacy process -- which I think is the purpose of the "Manage a legacy" appeals page. (Albeit it is really complex and hard to get to the heart of the matter with all the nuance stuff the business requires us to stuff on that page.)  

I'm just not sure of another page that would basically duplicate  what the "Manage a legacy appeals" page should be doing better -- but for stakeholder reasons, could be better. (Therefore, why would the new page be any different, since we'd still need stakeholder input.) I'm just not sure...

**Comment by mnorthuis** [comment]
Valid points @jenniferlee-dsva.  That said, this legacy appeals page sort of needs to be the landing page for anything legacy appeals related, so it just needs to be very clear that if you are looking to request a new appeal or NOD, this page tells them that's not possible and directs them to the new process. 

For this particular request, if a user searched for the NOD and we redirect them to this page, I just want to make sure they can quickly figure out that the NOD is not an option anymore, so even reference to it in the intro or where it says "If your decision was after...go to the decision review process"  we could add something that tells them that an NOD is not needed for this new process?

**Comment by jenniferlee-dsva** [comment]
I think the intro already tells ppl that new appeals are no longer accepted. So maybe the only other explicit clarification is to include Laura's proposed language about NODs no longer being valid? 

Laura if you're able to get this cleared -- without the SME blowing up the intro para into an essay -- please recommend proposed change to them and let them know it is to help clarity for those users who are still searching on NOD form. (We don't do this for most forms - but the NOD form and form detail page happen to get a very high search, on our site and organic. )

The stakeholder of the legacy appeals content is Paul Shute in disability compensation, and I would CC Fred, since he's the main technical SME on the new appeals/DR process.

**Comment by mnorthuis** [comment]
Are we waiting for a response from stakeholders on this one, or do we have ok to move forward with the redirect?

I believe the direction we are planning to go is to redirect this form to the legacy appeals page.  If the below is accurate, then we can update the top description of this issue and assign to dev. 

Current URL | Redirect Destination (new URL)
-- | --
http://www.vba.va.gov/pubs/forms/VBA-21-0958-ARE.pdf | https://www.va.gov/decision-reviews/legacy-appeals/
www.va.gov/vaforms/form_detail.asp?formno=21-0958 | https://www.va.gov/decision-reviews/legacy-appeals/

**Comment by jenniferlee-dsva** [comment]
I think the only thing remaining was to determine the content update/s that would need to be made and where -- in order to confirm the manage legacy appeals page is where we want to redirect to. 

Which I think ^ content recommendation confirms. @lwalsh617 - Would you mind capturing the copy updates that you proposed ^ as a copy ticket and have that be ready in time for implementation of Mikki's redirects? (The redirects themselves will go through our normal implementation queue; but would be great if you could coordinate with Selina on the redirect email comms, so that she cc:s  you, Paul Shute, and Fred for awareness.)

**Comment by mnorthuis** [comment]
I have updated the issue description to accurately reflect the redirects that are needed.  

@lwalsh617 @lunascoop These redirects can be implemented when your team is ready.

**Comment by jenniferlee-dsva** [comment]
@lunascoop - I still don't see the redirects working for these 2 URLs. Can you follow up and see what the ETA is? 
fysa @johnhashva @stephorkandatx 

The NOD form is one of the highest searched VA forms, so we should def have redirects in place for the form and the form detail page.

**Comment by lunascoop** [comment]
Sorry, totally overlooked this. @ncksllvn @kelsonic since these seem to be legacy pages, I am sending the stakeholder comm now and the redirect is scheduled for **06/09**.

**Comment by kelsonic** [comment]
@lunascoop Thanks for the update! Sending up a PR for this now 🙂

**Comment by kelsonic** [comment]
For this redirect:

Current URL  |  Redirect Destination (new URL)
---  |  ---
http://www.vba.va.gov/pubs/forms/VBA-21-0958-ARE.pdf | https://www.va.gov/decision-reviews/legacy-appeals/

We will need to have the VBA team implement this redirect. Not sure who to contact about that request though CC @ncksllvn @lunascoop @stephorkandatx

**Comment by jenniferlee-dsva** [comment]
I replied via email re ^. 

> Hi Selina – Per Kelson’s ticket comment about not being able to redirect the PDF URL, could you reach back out to Sandy, Adrianne, and ask if they are able to implement the redirect (301 permanent redirect) for that one URL? 
> 
> It make sense we can’t implement it bc PDF’s are not pages with our header injection. Going forward, please cc Beth (VBA content lead) on VBA things; and John (redirects PO) on redirects. Thank you!

**Comment by lunascoop** [comment]
Hi @kelsonic you can implement this redirect now. No wait needed per an email I received from Jen this morning.

**Comment by kelsonic** [comment]
@lunascoop PR merged! 🎉

**Comment by lunascoop** [comment]
There is an issue w/ redirecting the forms page (https://www.va.gov/vaforms/form_detail.asp?formno=21-0958) to the new URL per this [Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1591026948005900). Will reach out to stakeholders to see if they can implement the redirect on their end.

**Comment by kelsonic** [comment]
This ticket is now blocked by https://github.com/department-of-veterans-affairs/va.gov-team/issues/9824, we will need to wait for said ticket to be completed before we are able to close out this one 🙂

**Comment by ncksllvn** [comment]
The DevOps repo now has support for this! 

As of this PR - https://github.com/department-of-veterans-affairs/devops/pull/7174

**Comment by lwalsh617** [comment]
@kelsonic -- I updated the redirect destination at the top of this ticket. Here it is as well: https://www.va.gov/decision-reviews/legacy-appeals/#a-legacy-appeal-follows-these-
(FYI, we changed it from the page url to an anchor tag url.)

**Comment by kelsonic** [comment]
PR is up! https://github.com/department-of-veterans-affairs/devops/pull/7215

**Comment by kelsonic** [comment]
This ticket is blocked and we need to wait for Taylor to come back to add further support for query param redirects, see more details here: https://dsva.slack.com/archives/CBU0KDSB1/p1594221248425400?thread_ts=1594151320.381900&cid=CBU0KDSB1

**Comment by kelsonic** [comment]
Woot! This ticket should not be implemented and is ready for validation + closing:

https://dsva.slack.com/archives/CBU0KDSB1/p1594151320381900

**Comment by MickinSahni** [comment]
Validated we're taken to the header on the decision-review page. Closing out this ticket
